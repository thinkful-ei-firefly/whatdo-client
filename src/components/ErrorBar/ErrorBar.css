.fetch-error-bar {
    font-family: sans-serif;
    margin: 10px auto;
    max-width: 800px;
    border-radius: 7px;
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    align-content: center;
    border-radius: 7px;
    /* height: 100%; */
    align-content: stretch;
    justify-content: space-around;
    text-shadow: 0px 0px 6px rgba(255, 255, 255, 1);
    -webkit-box-shadow: 0px 5px 40px -10px rgba(0, 0, 0, 0.57);
    -moz-box-shadow: 0px 5px 40px -10px rgba(0, 0, 0, 0.57);
    transition: all 0.4s ease 0s;
    padding: 10px;
    background-image: linear-gradient(to top, #ffffffda, #f71109da, #ffffffda);
  }
  
  .fetch-error-bar {
    text-align: center;
  }
  
  .fetch-error-bar h2 {
    width: 95%;
    color: darkred;
    padding: 10px;
    text-align: center;
    margin: 0 auto;
    border-radius: 7px;
    text-shadow: 0px 0px 6px rgba(253, 253, 253, 0.2);
    -webkit-box-shadow: 0px 5px 40px -10px rgba(0, 0, 0, 0.57);
    -moz-box-shadow: 0px 5px 40px -10px rgba(0, 0, 0, 0.57);
    transition: all 0.4s ease 0s;
    background: rgba(250, 248, 241, 0.666);
  }
  
  .error-bar-header {
    text-align: center;
  }
  