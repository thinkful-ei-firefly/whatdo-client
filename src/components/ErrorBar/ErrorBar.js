import React from 'react'
import './ErrorBar.css'

export default class ErrorBar extends React.Component {

  render () {
    return (
      <div className="fetch-error-bar">
        <h2 className="error-bar-header">We're sorry, the server is experiencing some difficulties.</h2>
        <h3>Please wait a few minutes or refresh the page and try again.</h3>
      </div>
    );
  }
}
