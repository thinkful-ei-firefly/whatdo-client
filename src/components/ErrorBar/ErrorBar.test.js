import React from 'react'
import ErrorBar from './ErrorBar'
import renderer from 'react-test-renderer'

describe('<ErrorBar />', () => {
  it('renders as expected', () => {
    const tree = renderer
      .create(
        <ErrorBar />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})