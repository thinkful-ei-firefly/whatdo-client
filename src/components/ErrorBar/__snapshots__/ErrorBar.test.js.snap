// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBar /> renders as expected 1`] = `
<div
  className="fetch-error-bar"
>
  <h2
    className="error-bar-header"
  >
    We're sorry, the server is experiencing some difficulties.
  </h2>
  <h3>
    Please wait a few minutes or refresh the page and try again.
  </h3>
</div>
`;
